lib/index.ts

```diff
@@ -80,6 +88,7 @@ export default function storyblokIntegration(
     useCustomApi: false,
     bridge: true,
     componentsDir: "src",
+    enableFallbackComponent: false,
```

Comment:
Why not `true` by default?

Reply:
I think throwing errors is actually more meaningful, as the error handling allows for more detail (see vite-plugin-storyblok-components.ts line 49 and StoryblokComponent.astro line 27). The fallback component should just be an option if you really don't want your environment to break if a component is not configured properly. What do you think?

Reply:
As we agreed internally, let's leave it like this (opt-in, disabled by default)